tests/acceptance/test_mender-artifact.py

```diff
@@ -31,7 +31,7 @@
 LAST_BUILD_VERSION = None
 
 # params is the versions we will test.
-@pytest.fixture(scope="function", params=[1, 2, 3])
+@pytest.fixture(scope="function", params=[2, 3])
```

Comment:
I haven't looked closely at the exact changes you made, but you could consider skipping or deleting this test, as this is already tested extensively on other branches, and is almost solely dependent on the mender-artifact branch, not the meta-mender branch.

Reply:
I just cherry-picked a commit because it was easier that editing the tests :)